tests/test_null.py

```diff
@@ -3,23 +3,70 @@
 import pytest
 from preprocessy.handlenullvalues import NullValuesHandler
 
+dataframe = pd.read_csv("datasets/encoding/test2.csv")
```

Comment:
Can you move `test2.csv` file to a new directory inside datasets. This will ensure that datasets for testing particular modules don't get mixed. Also, you can **rename** the file to something that mirrors its contents so that it becomes easier for people to get an idea of what data the file contains.

Reply:
The dataset actually is general data, not specific to handling null values module.
The dataset contains the distance between the capitals in India.

Reply:
Fine then I'll shift all the datasets to the root of datasets directory.